app/views/settings/profiles/show.html.haml

```diff
@@ -46,6 +46,14 @@
             = material_symbol 'delete'
             = t('generic.delete')
 
+  .fields-row
+    .fields-row__column.fields-row__column-12
```

Comment:
You might want to use column 6 here, or merge with the row for the file input to achieve better layout